extutils/imgproc/apng2png.py

```diff
@@ -0,0 +1,285 @@
+"""
+Module to extract the frames of APNG **without** constraint violation checking.
+
+.. note::
+    https://wiki.mozilla.org/APNG_Specification
+    https://github.com/eight04/pyAPNG
+    https://github.com/tonix0114/apng-splitter
+"""
+
+from abc import ABC
+import binascii
+from fractions import Fraction
+from enum import Enum
+from dataclasses import dataclass, field
+import io
+import struct
+from typing import Union, Generator, List, Tuple
+
+from PIL import Image
+
+__all__ = ["extract_frames"]
+
+png_signature = b"\x89\x50\x4e\x47\x0d\x0a\x1a\x0a"
```

Comment:
Codacy found an issue: [Constant name "png_signature" doesn't conform to UPPER_CASE naming style](https://app.codacy.com/manual/RaenonX/Jelly-Bot/pullRequest?prid=6015490)